test/jdk/java/util/stream/GatherersMapConcurrentTest.java

```diff
@@ -313,4 +316,60 @@ public void behavesAsExpectedWhenShortCircuited(ConcurrencyConfig cc) {
 
         assertEquals(expectedResult, result);
     }
+
+    @ParameterizedTest
+    @MethodSource("concurrencyConfigurations")
+    public void behavesAsExpectedWhenCallerIsInterrupted(ConcurrencyConfig cc) {
```

Comment:
It might be helpful for future maintainers to put a comment on the behaveAsExpectedXXX tests so that it's easier to figure out what they are testing.

Reply:
That's fair, I decided to improve the names of the methods in the follow-on commit I just pushed 👍